### PR TITLE
Fix ml.update_job Request

### PR DESCRIPTION
**Pull request by philkra**
as titled.

### CODE REVIEW

**Review by delvedor — CHANGES_REQUESTED** [review]
LGTM, can you fix the conflicts and regenerate the output?

**Comment by philkra** [comment]
@delvedor conflicts resolved

File: specification/__common/ml/ModelPlotConfigEnabled.ts [hunk 1]
```diff
@@ -17,7 +17,22 @@
...
+   * If true, enables calculation and storage of the model bounds for each entity that is being analyzed.
+   */
+  enabled: boolean
+  /**
+   * @since 7.9.0
+   * @server_default true
+   */
+  annotations_enabled?: boolean
+  /**
+   * stability: experimental
```
Comment (delvedor) [inline specification/__common/ml/ModelPlotConfigEnabled.ts:34]: Should we add a `stability` tag for properties as well?

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**